README.md

```diff
@@ -100,21 +146,31 @@ PUT /registrations
 
 ### AWS Lambda
 
-* On the Lambda dashboard, choose "Create Function" and "Author From Scratch". Pick a suitable function name, at least Node.js 12.x, and create or reuse a role with basic Lambda execution rights. 
-* Copy the two files from this repository named `index.js` and `elasticsearch.js` into the Lambda workspace. Ensure the field "Handler" is set to `index.handler`.
-* Under "Environment Variables", create a new variable `ELASTICSEARCH_DOMAIN` with the Elasticsearch "Endpoint" name we got above.
-* Under "Basic Settings", the default 128 MiB RAM and 3 second timeout should be sufficient for testing.
-* At the top, under "Designer", start the "Trigger" template and choose "API Gateway".
+- Search for and open the **Lambda** dashboard.
+- Choose **Create Function** and **Author From Scratch**. 
+- Pick a suitable function name (e.g. pass_pelion_data)
+- Choose at least Node.js 12.x 
+- Select *Create a new role with basic Lambda permissions* or *Use an existing role* if you already have one. 
+- Click **Create Function**
+- Copy the contents of `index.js` from this repository into the file named `index.js` within the Lambda Function code workspace.
+- Create a new file in the Lambda workspace named `elasticsearch.js` and copy the contents from the file `elasticsearch.js` from this repository into it. 
+- Under *Runtime settings*, ensure the field **Handler** is set to **index.handler**.
+- Under **Environment Variables**, create a new variable `ELASTICSEARCH_DOMAIN` with the Elasticsearch *Endpoint* name from earlier without the "https://". 
+- Under **Basic Settings**, ensure the default 128 MiB RAM and 3 second timeout are set. This should be sufficient for testing.
+- At the top, under **Designer**, click the **Add Trigger** template and choose **API Gateway**.
 
 ### AWS API Gateway
 
-* In the "Add trigger" template, create a new "REST API" with "Open with API key" under security. 
-* Enable logging. Note, this may fail due to missing permissions. Follow the guid above about CloudWatch roles and try again.
-* Note the "API Endpoint" URL and the "API key".
+- In the **Add trigger** template, choose **Create an API** then select **REST API**.
+- Under *Security*, choose **API key**
+- Enable logging. Note, this may fail due to missing permissions. Follow the guide above about CloudWatch roles and try again.
+- Note the **API Endpoint** URL and the **API key**.
 
 ### Pelion Cloud REST API
 
-From the Arm Pelion Device Management portal, create a new API key. From a command terminal execute:
+- Log in to the [Pelion Device Management portal](https://portal.mbedcloud.com/login).
```

Comment:
Maybe add a bit of an intro here as to why these REST call are necessary.  Something like:

Now that your AWS account is configured to receive the notification messages from Pelion, the call back URL must be set on your Pelion account and subscriptions set to start the message flow.

Reply:
good idea. added more context in 4b755d7.